README.md

```diff
@@ -5,6 +5,27 @@ the
 user like authenticating the users, issuing the access/refresh tokens, api keys, validating the permissions for the
 users.
 
+## System flow diagram
+
+Below diagram depicts the system flow for Authentication service
+
+```mermaid
+graph LR;
+    A(Client) --->|Authenticate| B(Gateway) 
+    B -->|Authenticate| C(Auth Service) 
+    C -->|Authenticate| D(Keycloak Service)
+    C -->|Authorize| E(TDEI Database)
+```
+
+- Auth service is responsible for Authenticating and Authorizing the HTTP requests. Also, verifies the access token,

```

Comment:
access token from/of?

Reply:
In the previous "readme". all the components of mermaid diagram when written down, you used quotes around them. 
Example `Data Service` etc. Here, I don't see it. Is this intended?